artemis/reporting/modules/sql_injection_detector/template_expose_to_sql_injection.jinja2

```diff
@@ -0,0 +1,69 @@
+{% if "exposed_to_sql_injection" in data.contains_type %}
+    <li>{% trans %}We identified that the following URLs are exposed to SQL Injection or Time Base SQL Injection:{% endtrans %}
+        <ul>
+            {% for report in data.reports %}
+                {% if report.report_type == "exposed_to_sql_injection" %}
+                    <li>
+                        {{ report.target }}
+                        <ul>
+                        {% for message in report.additional_data.result %}
```

Comment:
you may use report.additional_data to pass structured data, parsing strings is not a good idea as it's brittle